Sources/Basics/Observability.swift

```diff
@@ -161,6 +162,14 @@ public protocol DiagnosticsHandler: Sendable {
     func handleDiagnostic(scope: ObservabilityScope, diagnostic: Diagnostic)
 }
 
+//helper protocol to configurate the style of diagnostics
```

Comment:
Please use doc comments formatting to make these more discoverable.

```suggestion
/// Helper protocol to configurate the style of diagnostics
```

Reply:
Having another look at it, does this even warrant a separate protocol if there's only one implementation of it?